lib/config.ex

```diff
@@ -5,6 +5,19 @@ defmodule Xcribe.Config do
 
   @valid_formats [:api_blueprint, :swagger]
 
+  @doc """
+  Return if serve mode is enable.
+
+  If no config was given the default is false.
+
+  To configure serve mode:
```

Comment:
I was thinking serve here have the intention to mean `servidor` or `servir`?

Reply:
He probably meant `To configure server mode:`

Reply:
Fixed :+1: